mlir/utils/jenkins/Jenkinsfile.downstream

```diff
@@ -1,4 +1,12 @@
 // used for public CI
+
+import groovy.transform.Field
```

Comment:
Jenkins is complaining that it doesn't allow ConcurrentHashMap for external CI. I am going to push a change to undo this